src/components/TodoList/TodoList.tsx

```diff
@@ -2,0 +1,25 @@
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../App';
+import { User } from '../../App';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo, index) => {
+        const { title, userId } = todo;
+        const currentUser: User = (usersFromServer || []).find(
+          user => user.id === userId,
+        )!;
+        const moddedTodo = { ...todo, user: currentUser };
+
+        return <TodoInfo todo={moddedTodo} key={`${title}-${index}`} />;
```

Comment:
Using the `title` and `index` as the key for each `TodoInfo` component might lead to issues if there are duplicate titles. Consider using a unique identifier like `todo.id` for the key to ensure each element is uniquely identified.